README.md

```diff
@@ -1,3 +1,12 @@
+### Table Of Contents
```

Comment:
Can we move this above "Installation" and below the "About" section? And remove the "About Gidari" link.

Reply:
I have made the required changes. You can review them.

Reply:
I have made the required changes. You can review them accordingly.